pomodoro/src/main/resources/static/scripts/app.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+angular.module('pomodoroapp', ['ngRoute'])
+.config(["$routeProvider", "$locationProvider", function($routeProvider, $locationProvider) {
```

Comment:
Please apply the replacement of tab character with 2 spaces in JS files as well.